vr-wg-charter.html

```diff
@@ -171,20 +169,20 @@ <h2>
         </h2>
         <p>
           With the rising availability of a wide range of head-mounted
-          displays, Virtual Reality (VR) has recently emerged as a critical
-          field of evolution for human-machine interactions.
+          displays as well as platform APIs for environment sensing,
+          augmented and virtual reality (collectively known as XR) has recently emerged as a critical field of evolution for human-machine interactions.
         </p>
         <p>
           The Web provides a promising preexisting ecosystem for the creation,
-          distribution, and experiencing of VR content, applications, and
+          distribution, and experiencing of XR content, applications, and
           services.
         </p>
         <p>
           As explored during the <a href=
           "https://www.w3.org/2016/06/vr-workshop/report.html">October 2016 W3C
           Workshop</a>, a first step in exploiting that potential relies on
           allowing browsers to display and react to content adapted to
-          available head-mounted displays. The <a href=
+          available head-mounted displays and to handheld devices in "magic window" mode. The <a href=
           "https://www.w3.org/community/webvr/">Community-Group incubated</a>
           <a href="https://w3c.github.io/webvr/">WebVR API</a> has already
```

Comment:
Do we want to say instead (should be a stable enough URL to reference at this time):

`<a href="https://immersive-web.github.io/webxr/">WebXR Device API</a>`

Or explicitly refer to the WebVR API 1.1 in which case we should link to https://immersive-web.github.io/webvr/spec/1.1/ for clarity?

Reply:
I didn't update the URLs because we're probably changing the slug that W3C uses to reflect the name change.